docs/root/api-docs/xds_protocol.rst

```diff
@@ -678,6 +678,17 @@ server does not provide EDS/RDS responses, Envoy will not initialize
 itself during the initialization phase and the updates sent via CDS/LDS
 will not take effect until EDS/RDS responses are supplied.
 
+.. note::
+
+   Envoy specific implementation note: Warming of ``Cluster`` is completed only
+   when a new ``ClusterLoadAssignment`` response is supplied by management server
+   even if there is no change in endpoints. 
+   Warming of ``Listener`` is completed even if management server does not send a 
+   response for ``RouteConfiguration`` referenced by ``Listener``. Envoy will use the
+   previously sent ``RouteConfiguration`` to finish ``Listener`` warming. Management Server
+   has to send the ``RouteConfiguration`` response only if it has changed or it was never 
+   sent in the past.
```

Comment:
nit: split this into 2 notes, or a bulleted list of 2 notes. It's not super clear which parts are related.